.github/workflows/python-app.yml

```
@@ -9,7 +9,11 @@ name: Unit Tests
 
 on:
   push:
+<<<<<<< HEAD
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)